csharp/src/Ice/SslTransceiver.cs

```diff
@@ -552,32 +552,12 @@ private void FinishAuthenticate()
                 bool certificateNameMismatch = (errors & (int)SslPolicyErrors.RemoteCertificateNameMismatch) > 0;
```

Comment:
Remove the certificateNameMismatch boolean which should no longer be used in the if block below.